web/modules/custom/dpl_event/src/PriceFormatter.php

```diff
@@ -35,10 +70,7 @@ public function formatPrice(string $price_string): string {
       // Format the numeric part of the price using formatRawPrice.
       $formatted_price = $this->formatRawPrice($price_string);
```

Comment:
This is a bit off topic because it is not a part of this PR. But I do not understand the concept of `formatRawPrice. If we are dealing solely with numbers I think the input and the return types should reflect that. Either with integers or float.
Another thing that I am thinking of (if I dare to say it :)) is why not using a library for handling prices? Are there any special circumstances that prevents that?

Reply:
The `raw` might mislead you a bit;
It's not a raw number, but a formatted number. E.g. 1000000 becomes 1,000,000 in danish.
The reason we call it raw here, is because it doesnt have the "DKK".

I'm not the original author of the file, so I cant tell you why we arent using a library - but, I would imagine that there's a good reason :)